doc/integrator/ngeo.rst.tmpl

```diff
@@ -175,6 +197,10 @@ for the HTML.
 Extending or overriding an existing component
 ----------------------------------------------
 
+.. note::
```

Comment:
There are so many points saying "not possible in the simple application mode" that I wonder if we should create a table instead:
containing list of things that can be done, and then a checkbox "available in simple application mode yes/no"

Reply:
I think no, for me, we should see that when we read the concerned documentation...